autogenerated.ts

```diff
@@ -1,12 +1,14 @@
 // Example of code autogenerated by editor extension.
 // Auto-generated. Do not edit.
-enum MlRunnerLabels {
-  //% block="Shake"
-  Shake = 1,
-  //% block="Still"
-  Still = 2,
-  //% block="Circle"
-  Circle = 3,
+namespace mlrunner {
+  export namespace Action {
```

Comment:
I've added a nested namespace to make the Javascript code more readable and, since this is user defined, it avoids accidental overshadowing of existing variables/functions in the top `mlrunner` namespace.